### PR TITLE
clarify docs for disk group when moving allocations.

### PR DESCRIPTION
**Pull request by jasonwalker80**

### CODE REVIEW

**Comment by chrisamiller** [comment]
+1 - appreciated!

File: lib/perl/Genome/Model/Build/Command/MoveAllocations.pm [hunk 1]
```diff
@@ -19,7 +19,7 @@ class Genome::Model::Build::Command::MoveAllocations {
         disk_group => {
             is                  => 'Genome::Disk::Group',
             is_optional         => 1,
-            doc => 'A specific disk group to move all data to (defaults to AnP configured location)',
+            doc => 'A specific disk group to move all data to. Defaults to AnP configured location unless disk_group_models is already defined in the users environment',
```
Comment (tmooney) [inline lib/perl/Genome/Model/Build/Command/MoveAllocations.pm]: It might be nice if this were rendered as "user's".

Comment (jasonwalker80) [inline lib/perl/Genome/Model/Build/Command/MoveAllocations.pm]: will it work to escape it? `user\'s`?

Comment (jasonwalker80) [inline lib/perl/Genome/Model/Build/Command/MoveAllocations.pm]: yep, that works

Comment (tmooney) [inline lib/perl/Genome/Model/Build/Command/MoveAllocations.pm]: It seems to work in a quick test!  Could also use `q{}` or the like instead.

**Pull request merged**